src/modules/content-s3/content-s3.c

```diff
@@ -59,14 +111,10 @@ static int parse_config (struct content_s3 *ctx,
     const char *uri = NULL;
     const char *bucket = NULL;
     const char *cred_file = NULL;
+    int is_virtual_host = 0;
```

Comment:
I think this function would be a little clearer if it just allocated and returned a `struct s3_config *`.  Then you wouldn't need all the tmp variables.

Reply:
In current update, `parse_config()` still updates a config, doesn't allocate it.  Any reason this was not implemented?